gi-gtk-hs/travis/build.sh

```diff
@@ -1,4 +1,6 @@
-#!/bin/bash -ex
+#!/usr/bin/env bash
+
+set -ex
```

Comment:
Was there a reason to add the flags to the shebang in this case? If so, I'm happy to undo that part.

Reply:
Not sure (I inherited this code), but in any case we are not using travis any more, so this will not make a difference I think.